src/pages/Books.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Book from '../components/Book';
+
+const Books = () => {
+  return (
+    <div className="books">
+      <ul>
+        {Array(5)
+          .fill('')
+          .map((book, index) => {
+            return <Book name={index} />;
+          })}
```

Comment:
- [ ] **Required:** I see you have written the component for for the form but probably forgot🥵  to import and render it on the book page. It is required to have a form, kindly include it.